external-dns.yaml

```diff
@@ -37,6 +37,26 @@ pipeline:
 
   - uses: strip
 
+subpackages:
+  - name: ${{package.name}}-bitnami-compat
+    dependencies:
+      provides:
+        - mysql-bitnami-compat=${{package.full-version}}
```

Comment:
```suggestion

```
I think we can remove this given it's not version streamed. also mysql (hehe, get over it now)